core/src/test/java/org/elasticsearch/tribe/TribeServiceTests.java

```diff
@@ -183,6 +188,28 @@ public void testMergeCustomMetaDataFromMany() {
         assertEquals(mergedCustom.getData(), "data2"+String.valueOf(n));
     }
 
+    public void testTribeNodeDeprecation() throws IOException {
+        final Path tempDir = createTempDir();
+        Settings.Builder settings = Settings.builder()
+            .put("node.name", "test-node")
+            .put("path.home", tempDir)
+            .put(NetworkModule.HTTP_ENABLED.getKey(), false)
+            .put(NetworkModule.TRANSPORT_TYPE_SETTING.getKey(), "mock-socket-network");
+
+        boolean tribeServiceEnable = randomBoolean();
+        if (tribeServiceEnable) {
+            String clusterName = InternalTestCluster.clusterName("single-node-cluster", randomLong());
```

Comment:
We do not need this, a non-random cluster name is fine and that utility method is really for the internal clusters that are created in integration tests.